data/noiseprofiles.json

```diff
@@ -8527,6 +8550,50 @@
+            {"name": "D-Lux 7 iso 1250", "iso": 1250, "a": [4.52774965368715e-05, 1.51100949333751e-05, 3.14811536226338e-05], "b": [1.28568722374818e-07, 6.50704380383652e-08, 1.08320799150238e-07]},
+            {"name": "D-Lux 7 iso 1600", "iso": 1600, "a": [5.66935183440472e-05, 1.91534593194069e-05, 3.99447413478979e-05], "b": [1.8331060328747e-07, 7.93452648376039e-08, 1.39163479907587e-07]},
+            {"name": "D-Lux 7 iso 2000", "iso": 2000, "a": [7.16429205278605e-05, 2.38068607862341e-05, 4.93084757397271e-05], "b": [1.77254313446102e-07, 1.07775297458744e-07, 1.88286039359388e-07]},
+            {"name": "D-Lux 7 iso 2500", "iso": 2500, "a": [8.96358008362269e-05, 2.99008737429642e-05, 6.28245830621012e-05], "b": [2.58204464086174e-07, 1.55739519185636e-07, 2.46228803288064e-07]},
+            {"name": "D-Lux 7 iso 3200", "iso": 3200, "a": [0.00011392317788068, 3.84404017480665e-05, 7.98345675644352e-05], "b": [4.45055526949849e-07, 2.20727509548738e-07, 4.06186881029347e-07]},
+            {"name": "D-Lux 7 iso 4000", "iso": 4000, "a": [0.000143709387802207, 4.79624945858196e-05, 9.89427655680384e-05], "b": [4.87475913124482e-07, 2.96619131233199e-07, 5.15772151127226e-07]},
+            {"name": "D-Lux 7 iso 5000", "iso": 5000, "a": [0.00018089384410813, 5.95980760283586e-05, 0.000125559367976104], "b": [5.10664537994349e-07, 3.93999328385326e-07, 5.7372939649651e-07]},
+            {"name": "D-Lux 7 iso 6400", "iso": 6400, "a": [0.000225831183408179, 7.45536170945211e-05, 0.000157209103689251], "b": [6.99474327302804e-07, 5.5731462820492e-07, 8.07360918835466e-07]},
+            {"name": "D-Lux 7 iso 8000", "iso": 8000, "a": [0.000284169087708228, 9.35599680814793e-05, 0.000197705268762258], "b": [1.06118168906003e-06, 8.58104641959915e-07, 1.23655278044128e-06]},
+            {"name": "D-Lux 7 iso 10000", "iso": 10000, "a": [0.00036243475124656, 0.000118653768298497, 0.000245138702803466], "b": [2.18094201528324e-06, 1.51197754569152e-06, 2.53047479139305e-06]},
+            {"name": "D-Lux 7 iso 12500", "iso": 12500, "a": [0.000449640776725114, 0.000148446440408457, 0.000311886091782767], "b": [2.37996955034241e-06, 2.00015896812575e-06, 2.88449986496872e-06]},
+            {"name": "D-Lux 7 iso 16000", "iso": 16000, "a": [0.000283067530749418, 9.35090443171114e-05, 0.00019806078346406], "b": [1.05913275268099e-06, 8.56211171321623e-07, 1.24206348899041e-06]},
+            {"name": "D-Lux 7 iso 20000", "iso": 20000, "a": [0.000357916782226216, 0.000118289345484384, 0.000249054892994411], "b": [1.59063650580094e-06, 1.29866798859529e-06, 1.8919995222713e-06]},
+            {"name": "D-Lux 7 iso 25000", "iso": 25000, "a": [0.000447900876269519, 0.000148891527550444, 0.000312481699207148], "b": [2.39306717704209e-06, 1.97777706381915e-06, 2.90082699795567e-06]}
+          ]
+        },
+        {
+          "comment": "leica m10 contributed by mandre",
+          "model": "LEICA M10",
```

Comment:
Don't think this one'll work: there is no entry for the M10 in rawspeed/data/cameras.xml so maker/model are not "normalized" properly: original maker is "Leica Camera AG" in the Exif, and not translated to "Leica" expected here.

Reply:
I tested it on a raw file, and the noise profile is correctly detected

Reply:
Must be doing partial string matching on the maker then...

In any case, would be good to go the normalized rawspeed route and avoid the mix.